tests/calculations/test_geomopt.py

```diff
@@ -14,7 +14,7 @@
 
 
 def test_geomopt(fixture_sandbox, generate_calc_job, janus_code, model_folder):
-    """Test generating singlepoint calculation job"""
+    """Test generating geomopt calculation job"""
```

Comment:
Minor thing, but this shouldn't really be in this PR. We can probably let it go as it's not really functional code.